grafeas-v1/lib/grafeas/v1.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Auto-generated by gapic-generator-ruby. DO NOT EDIT!
+
+require "grafeas/v1/grafeas"
+require "grafeas/v1/version"
+
+module Grafeas
+  ##
+  # To load this package, including all its services, and instantiate a client:
+  #
+  #     require "grafeas/v1"
+  #     client = ::Grafeas::V1::Grafeas::Client.new
```

Comment:
Does this example still work without default credentials?

Reply:
It might, if the underlying implementation doesn't require credentials or has a different credentialing mechanism than Google's.